admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1392,6 +1392,44 @@ $product-page-padding-bottom: 80px !default;
   }
 }
 
+.product_packed_products {
+  .search-with-icon {
+    max-width: 377px;
+  }
+
+  .entities-list-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-right: -15px;
+    margin-left: -15px;
+  }
+
+  .packed-product {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    max-width: 180px;
+    margin: 0.5rem;
```

Comment:
```suggestion
    margin: 0.5rem;
    @include border-radius($border-radius)
```

It could be different but you've got the idea behind

Reply:
the border is not even present in the mockup so we'll have to remove it and the radius won't be visible either so I don't think it's worth changing this one